activiti-cloud-services-audit-jpa/src/main/resources/META-INF/application.properties

```diff
@@ -0,0 +1,2 @@
+# enable clean up endpoints
+activiti.enable-clean-up=true
```

Comment:
Same as my comment on [query-service](https://github.com/Activiti/activiti-cloud-query-service/pull/196#discussion_r262568897): it cannot be inside `application.properties`, the file should have another name and it should be referenced by an auto-configuration. Maybe this auto-configuration can live inside a new maven module `activiti-cloud-services-rest-conf` inside service-common repository so it's shared by runtime bundle, query and audit.